src/controlplane/client/connector/connector.go

```diff
@@ -30,27 +30,47 @@ type ConnParams struct {
 	Client client.HTTPDoer
 }
 
-type defaultConnector struct {
-	// TODO: Use a non-sdk logger
-	logger        log.Logger
-	authenticator authenticator.Authenticator
-	endpoints     []config.Endpoint
+type Config struct {
```

Comment:
could we add some comments since they are exported?

Reply:
i think the commets will add more noice in this case, i think the variable names are clear self explained wdyt?

Reply:
mmmm true :)